### PR TITLE
make author and maintainers field consistent accross packages

### PR DESCRIPTION
**Pull request by Elhebert**
### What does it do?

Make the author and maintainers field consistent across all our packages.
The `name` property was updated to match the name used in our licenses.

https://github.com/strapi/strapi/blob/615715b2c11bf5cd4ca0742cb36b491e622d525e/LICENSE#L1

### CODE REVIEW

File: packages/providers/upload-cloudinary/package.json [hunk 1]
```diff
@@ -17,13 +17,13 @@
   },
   "license": "SEE LICENSE IN LICENSE",
   "author": {
-    "name": "Luca Perret",
-    "email": "perret.luca@gmail.com",
-    "url": "https://github.com/lucaperret"
+    "name": "Strapi Solutions SAS",
```
Comment (alexandrebodin) [inline packages/providers/upload-cloudinary/package.json:20]: you can keep the original author here

Comment (Elhebert) [reply packages/providers/upload-cloudinary/package.json]: Copy/paste gone wrong. Fixed it alongside a small adjustment in another provider to keep the same formatting.

**Review by petersg83 — APPROVED** [review]
LGTM

**Review by alexandrebodin — APPROVED** [review]
LGTM

**Pull request merged**